flink-metrics/flink-metrics-core/src/main/java/org/apache/flink/metrics/MeterView.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.metrics;
+
+/**
+ * A MeterView provides a rate of events per second over a given time period. The events are counted by a {@link Counter}.
+ * A history of measurements is maintained from which the rate of events is calculated on demand.
+ */
+public class MeterView implements Meter, View {
```

Comment:
I think it would be good to add some more javadocs to this class, at least for the `timeSpanInSeconds` argument of the ctor.